pyccel/ast/datatypes.py

```diff
@@ -221,8 +220,8 @@ def codomain(self):
                   'str': String}
 
 
-class UnionType(Basic):
-    _attribute_nodes = ()
+class UnionType:
```

Comment:
@saidctb I have a Codacy error here:
> Missing class docstring

I'm not clear sure what this class is for so I'm not sure how to write the docstring. Can you provide any insight?

Reply:
I had a go myself